### PR TITLE
Proofreading of metabolomics tutorial

### PR DESCRIPTION
**Pull request by simonbray**
@bebatut asked me to do some proofreading yesterday. I picked the metabolomics tutorial to start with, because it seemed like an interesting topic and I hoped I could learn something new at the same time. Here are the results. Overall nice tutorial, clear to follow. :)
ping @foellmelanie @lecorguille

### CODE REVIEW

File: topics/metabolomics/slides/introduction.html [hunk 1]
```diff
@@ -58,16 +58,15 @@
...
-  - Common physico-chemical propreties
+  - DNA/RNA: 4 bases
+  - Proteins: 20 amino acids
+  - Common physico-chemical properties
   - Extraction/Analysis: easy automation
 
 - Metabolites
-  - Number ≥ 150 000 in Nature
-  - Most of them not identified
+  - Number ≥ 150 000 in Nature; most not identified
```
Comment (simonbray) [inline topics/metabolomics/slides/introduction.html]: just changes this so all the points would fit on the slide

File: topics/metabolomics/tutorials/msi-finding-nglycans/tutorial.md [hunk 2]
```diff
@@ -157,27 +157,27 @@ To get an overview of the combined dataset we will use the **MSI Qualitycontrol*
...
...ure plot of the treated kidney roughly corresponds to the morphology of the kidney while in the control file there are no visible kidney structures. The first principal componant in the PCA analysis roughly differentiates the spectra of the two subfiles which is a good sign for finding differences in the treated vs. control kidney files. For the following preprocessing steps, it is important to know the rough number of data points that one peak consists of. From page 27 on, the average mass spectra of the two glycan calibrants are shown. Due to our harsh resampling to 0.1 m/z, the calibrant 1257.4225 (Man5GlcNAc2) shows quite an angular peak shape. The complete peak consists of about 6 data points, which corresponds to a width of 0.5 m/z. On page 28 the average spectrum for each subfile is plotted, showing a nice peak in the treated subfile and only basal intensity in the control subfile. Page 29 and 30 show the average mass spectra for the 1542.5551 (Man3GlcNAc5) calibrant. The intensities of this calibrant are relatively low and only slightly higher in the treated than in the control subfile. Furthermore, the split up peak shape is not useful for defining the peak width properly.
```
Comment (simonbray) [inline topics/metabolomics/tutorials/msi-finding-nglycans/tutorial.md]: I didn't find this paragraph so clear. In the third sentence, do you want to say 'do represent' or 'do not'?
Also, what do the page numbers refer to? I guess the PDF that is produced by the MSI quality control tool, but this seems to have less than 27 pages.

File: topics/metabolomics/tutorials/msi-finding-nglycans/tutorial.md [hunk 3]
```diff
@@ -35,21 +35,21 @@ contributors:
...
 
-In this tutorial we will determine and identify N-linked glycans by differential analysis of PNGase F treated and control murine kidney sections. The data analysis steps can be transferred to any other application that requires finding differential analytes such as for biomarker discovery. 
+In this tutorial we will determine and identify N-linked glycans by differential analysis of a control murine kidney sections with a section treated with PNGase F. The data analysis steps can be transferred to any other application that requires finding differential analytes, such as for biomarker discovery.
```
Comment (foellmelanie) [inline topics/metabolomics/tutorials/msi-finding-nglycans/tutorial.md]: thanks a lot @simonbray for the corrections! Here I think it should be: of a control murine kidney section (without 's')?.

Comment (simonbray) [reply topics/metabolomics/tutorials/msi-finding-nglycans/tutorial.md]: Yes, thanks for catching that.

File: topics/metabolomics/tutorials/msi-finding-nglycans/tutorial.md (hunk 2 above)
Comment (simonbray) [reply topics/metabolomics/tutorials/msi-finding-nglycans/tutorial.md]: Could one of you maybe comment on this @lecorguille @foellmelanie ? The page numbers in particular are strange. When I tested the tool I got a PDF with only 17 pages.

Comment (foellmelanie) [reply topics/metabolomics/tutorials/msi-finding-nglycans/tutorial.md]: @simonbray Sorry I was called away during reviewing and did not manage to continue. You are right, it should be 'do not represent'. You probably you tested the training with the workflow? There is a connection missing between the calibrants and the quality control. With the calibrant inputs the PDF has 35 pages. Sorry,  I will update the workflow very soon. 
Thanks a lot for testing and reviewing! :+1:

Comment (simonbray) [reply topics/metabolomics/tutorials/msi-finding-nglycans/tutorial.md]: You're welcome! Yes, you're right, I missed that step.

**Comment by bebatut** [comment]
@foellmelanie do you think it is ready to be merged?

**Comment by foellmelanie** [comment]
@bebatut  I am ok with merging, thanks for asking

**Comment by bebatut** [comment]
Thanks a lot @simonbray and @foellmelanie

**Pull request merged**